src/sentry/static/sentry/app/views/eventsV2/table/cellAction.tsx

```diff
@@ -240,6 +248,14 @@ class CellAction extends React.Component<Props, State> {
           {t('Go to summary')}
         </ActionItem>
       );
+
+      const KeyTransactionRenderer = actionRenderers[Actions.TOGGLE_KEY_TRANSACTION];
```

Comment:
alternatively you can do `actionRenderers?.[Actions.TOGGLE_KEY_TRANSACTION]` and drop the default object above